test/e2e/tests/test_sagemaker_endpoint_autoscaling.py

```diff
@@ -127,6 +127,23 @@ def generate_sagemaker_policy(generate_sagemaker_target):
 @service_marker
 @pytest.mark.canary
 class TestSageMakerEndpointAutoscaling:
+    def wait_until_update(
+        self, reference, previous_modified_time, wait_period=30, wait_time=5
+    ):
+        for i in range(wait_period):
```

Comment:
should we switch the wait_periods to 2 and wait_time to 30 or less

30 times every 5 sec seems unnecessary

Reply:
sg

Reply:
done.